core/types/shard_state.go

```diff
@@ -11,10 +12,13 @@ import (
 // ShardState is the collection of all committees
 type ShardState []Committee
 
+// BlsPublicKey defines the bls public key
+type BlsPublicKey [96]byte
+
 // NodeID represents node id (BLS address).
 type NodeID struct {
 	EcdsaAddress string
```

Comment:
may use Address type instead of string as well

Reply:
using fixed type is better than using string. Easier to debug.